packages/lambdas/src/shared/capture.ts

```diff
@@ -15,7 +15,7 @@ export const capture = {
    *
    * @param tracesSampleRate Sample rate to determine trace sampling.
    */
-  init: (tracesSampleRate = 1.0): void => {
+  init: (tracesSampleRate = 0.5): void => {
```

Comment:
<img width="729" alt="image" src="https://github.com/metriport/metriport/assets/2132564/3adf3392-4fda-482c-bff5-1dfc65973b94">

<img width="1756" alt="image" src="https://github.com/metriport/metriport/assets/2132564/d06ff334-f704-4d14-a18a-d21e46a84541">

https://docs.sentry.io/product/performance/transaction-summary/#what-is-a-transaction